backend/dal/requestDAL.go

```diff
@@ -33,3 +34,28 @@ func (dal *RequestDAL) GetLastRequests(before time.Time, size int) ([]models.Req
 
 	return reqs, nil
 }
+
+// InsertRequest Receives a request as a parameter and inserts into the database
+func (dal *RequestDAL) InsertRequest(request models.Request) (models.Request, error) {
+	insertQuery := `INSERT INTO request (name, description, extrainfo, minprice, maxprice, userid, categoryid, timestamp) 
+						VALUES ($1, $2, $3, $4, $5, $6, $7, $8)`
+
+	// Gets the controller of the database and executes the query
+	fmt.Println(request.Name)
+	fmt.Println(request.Description)
+	fmt.Println(request.ExtraInfo)
+	fmt.Println(request.MinPrice)
+	fmt.Println(request.MaxPrice)
+	fmt.Println(request.Userid)
+	fmt.Println(request.Categoryid)
+	fmt.Println(request.Timestamp)
+
+	_, err := dal.db.Exec(insertQuery, request.Name, request.Description, request.ExtraInfo, request.MinPrice, request.MaxPrice, request.Userid, request.Categoryid, request.Timestamp)
+
+	// Checks if any error happened during the query execution
+	if err != nil {
+		return request, err
```

Comment:
Maybe it'd be better to return just an empty request in this case models.Request{}